### PR TITLE
<Badge /> - Add UniDriver

### PR DESCRIPTION
**Pull request by Lukermelesh**

### CODE REVIEW

File: src/Badge/Badge.spec.js [hunk 1]
```diff
@@ -1,113 +1,130 @@
...
 
-    it('should not have any icons by default', () => {
-      const wrapper = createDriver(<Badge>Hello</Badge>);
-      expect(wrapper.getPrefixIcon()).toBeFalsy();
-      expect(wrapper.getSuffixIcon()).toBeFalsy();
-    });
+    describe('children prop', () => {
+      it('should render the text given as a children prop', async () => {
+        const driver = createDriver(<Badge>Hello</Badge>);
+        debugger;
```
Comment (mosheKerbel) [inline src/Badge/Badge.spec.js]: weird that eslint didn't catch it

Comment (Lukermelesh) [reply src/Badge/Badge.spec.js]: oops

**Comment by mosheKerbel** [comment]
looks good, need to fix conflict before merging.

**Comment by mosheKerbel** [comment]
merged, build faild due to 1 unrelated flaky timeout test

**Pull request merged**